src/components/AccountDetailsPanel.vue

```diff
@@ -21,7 +21,7 @@
       class="account-details-panel__not-existent">
       The account has never been seen in the network.
       <br>
-      Details will be displayed after the first interaction with this account.
+      Details will be displayed once the first transaction from the account is sent.
```

Comment:
```suggestion
      Details will be displayed once the account is involved in a transaction.
```

I think it's better because the account could also be the recipient and not the sender.

Reply:
ok, accepted